lib/storage/metadata/mongoclient/MongoClientInterface.js

```diff
@@ -440,6 +440,7 @@ class MongoClientInterface {
                 }
                 return cb(null, `{"versionId": "${objVal.versionId}"}`);
             });
+            return null;
```

Comment:
Is the caller not expecting the `cb` to be invoked?

Reply:
Yeah I don't understand how this solves the issue.